src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/InboundChannelAdapterLifecycleHandlerService.kt

```diff
@@ -61,15 +75,14 @@ class InboundChannelAdapterLifecycleHandlerService(
 
   /** Retrieve all InboundChannelAdapter on which perform commands */
   fun findInboundChannelAdapterBeans() =
-    applicationContext
-      .getBeansWithAnnotation(InboundChannelAdapter::class.java)
-      .filterNot { it.value is RedisStreamMessageSource }
-      .keys
+    applicationContext.getBeansWithAnnotation(InboundChannelAdapter::class.java).keys
 
-  fun invokeCommandForRedisStreamMessageSource(command: String) {
-    val controllerBusMessage =
-      MessageBuilder.createMessage(
-        "@eventDispatcherReceiverCommandChannelEndpoint.$command()", MessageHeaders(mapOf()))
-    controlBusInput.send(controllerBusMessage)
+  /**
+   * Determine if a bean name is related to Redis stream components. These are now handled via
+   * reactive approach and should be excluded.
+   */
+  private fun isBeanNameRedisRelated(beanName: String): Boolean {
+    return beanName.lowercase().contains("redis") ||
+      applicationContext.getBeanNamesForType(RedisStreamConsumer::class.java).contains(beanName)
```

Comment:
to be deleted.
why adding this logic here? we are searching for all InboundChannelAdapter beans, RedisStream is not an InboundChannelAdapter